BUILD.bazel

```diff
@@ -1,6 +1,174 @@
-exports_files(["staging/kvstore_state.json5"])
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+load(":rules.bzl", "basic_naming")
 
-exports_files(["staging/ledger_state.json5"])
+# Visibility for Docker image
+package(default_visibility = [
+    "//src/many-abci:__pkg__",
+    "//src/many-kvstore:__pkg__",
+    "//src/many-ledger:__pkg__",
+])
 
 # Required for Vergen
 exports_files([".git"])
+
+## PACKAGE CONFIGURATION ##
+# TODO: Get the version from the Cargo manifest
+VERSION = "0.1.0"
+
+# Exposes the value of the compilation mode to the package naming.
```

Comment:
Can you just add as a comment that this is used below for the `package_file_name` in the `pkg_tar` rule?